src/compass/utils/helpers.py

```diff
@@ -219,19 +223,27 @@ def _convert_to_utm(points_xy, epsg_src, epsg_dst):
     return list(zip(xt, yt))
 
 
-def burst_bbox_from_db(burst_id, burst_db_file=None, burst_db_conn=None):
+@functools.lru_cache(maxsize=2)
+def _read_burst_json(burst_db_file: Union[str, Path]) -> dict:
```

Comment:
If I remember correctly, @seongsujeong performed an analysis related to how much time we need to load into memory a burst data base allocated in SQlite or JSON. If I remember the conclusion correctly, we decided to go for Sqlite because of runtime performance. Can you cack up this choice with some numbers?

Reply:
I see. So summarizing what you are saying, we can provide the burst database (BDB) as a zipped JSON file. If we do so, we would have a flat 1-3 s to unzip the actual file and once the dictionary is loaded in memory, the other calls should be faster. Correct?

When  I was developing the stack processor, I remember experiencing some delays in scanning very big dictionaries. If I remember correctly, this was one of the analyses that also @seongsujeong (maybe, you can share your presentation in this PR, so that our memory will be refreshed).

I would not be too worried in including the time to download the BDB. For production 1) we do not use the stack processor 2) we assumed that the BDB is always staged somewhere and ready to use.

At the moment, the only implication I see in changing the file format of the BDB is how fast we are in scanning the all BDB for very large stacks (>100 acquisitions). Once we are sure, we are not introducing significant lags, I am fine with it

Reply:
I think seongsu's timings might have been checking what was the time to load *and* search for one item? We can check when he posts it.

The lookup in a dict, once it's loaded, should be in the microsecond range (since it's an O(1) operation) no matter how big the dict is.

Here's the lookup time for a 10M dict:
```python console
In [1]: import numpy as np
In [2]: large_dict = {k: k+1 for k in range(10_000_000)}
In [3]: idxs = (1e7 * np.random.rand(200)).astype(int)  # 200 random idxs from 0 to 1e7

In [4]: %timeit [large_dict[k] for k in idxs]
29 µs ± 932 ns per loop (mean ± std. dev. of 7 runs, 10,000 loops each)
```